app/supplejack/transformation/field_execution.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require 'webmock'
+
+module Transformation
+  # Only executes the code from the user
+  class FieldExecution
+    include WebMock::API
+
+    def initialize(field)
+      @field = field
+    end
+
+    # rubocop:disable Lint/UnusedBlockArgument
+    # rubocop:disable Security/Eval
+    # rubocop:disable Lint/RescueException
+    def execute(api_record)
+      WebMock.enable! unless Rails.env.test?
+      begin
+        block = ->(record) { eval(@field.block) }
+
+        @value = block.call(api_record)
+      rescue Exception => e
+        @error = e
+      end
+
+      WebMock.disable! unless Rails.env.test?
```

Comment:
hmm, I wonder when we `WebMock.enable!` if it will apply across everything, not only within the code we are running, which maybe could have side effects?

it might be good to add the `WebMock.disable` in a `ensure` block so we can ensure it always run?

Reply:
Thanks for your comment here bro. I think there is a risk of the webmock disable interfering with other running jobs. If that becomes an issue we will need to look at another way of preventing HTTP calls in the transformation :)